File: 실습/2장/01_function/Main.java
public class Main {

    public static int add(int a, int b) {
        return a + b;
    }

    public static void main(String[] args) {
        final String name = "Java";
        int count = 1;

        count = 2;

        System.out.println("언어: " + name);
        System.out.println("count: " + count);
        System.out.println("합계: " + add(10, 20));
    }
}

File: 실습/2장/01_function/Main.kt
fun add(a: Int, b: Int): Int {
    return a + b
}

fun main() {
    val name = "Kotlin"
    var count = 1

    count = 2

    println("언어: $name")
    println("count: $count")
    println("합계: ${add(10, 20)}")
}